syntaxes/fsh.tmLanguage.json

```diff
@@ -29,11 +29,91 @@
       "patterns": [
         {
           "name": "keyword.control.fsh",
-          "match": "\\b(Alias|CodeSystem|Expression|Extension|Description|Id|Instance|InstanceOf|Invariant|Logical|Mapping|Mixins|Parent|Profile|Resource|RuleSet|Severity|Source|Target|Title|Usage|ValueSet|XPath)(?=\\s*:)\\b"
+          "match": "\\b(Alias|Expression|Description|Mixins|Severity|Target|Title|Usage|XPath)(?=\\s*:)\\b"
+        },
+        {
+          "match": "\\b(CodeSystem|Extension|Id|Instance|InstanceOf|Invariant|Logical|Mapping|Parent|Profile|Resource|RuleSet|Source|ValueSet)\\s?(:)\\s+([A-Za-z0-9_-]+)\\b",
```

Comment:
This set of characters reminded me that you can also include `.` in some of the values after these keywords, so I think it would be good to include the `.` character with in the `([A-Za-z0-9_-]+)` set here.